lib/resque/scheduler/scheduling_extensions.rb

```diff
@@ -149,6 +174,77 @@ def remove_schedule(name)
 
       private
 
+      def supports_lua?
+        redis_master_version >= 2.5
+      end
+
+      def redis_master_version
+        Resque.redis.info['redis_version'].to_f
+      end
+
+      def startup_sha(refresh = false)
```

Comment:
The reason it removes the non-persisted schedules is that the schedules are not persisted so we dont wanna load them into our scheduler instance.

This is ugly and unnecessary, non-persisted schedules should only be in memory